pkgs/development/tools/profiling/gprof2dot/default.nix

```diff
@@ -0,0 +1,18 @@
+{ stdenv, fetchgit, pythonPackages }:
+
+pythonPackages.buildPythonPackage {
+  name = "gprof2dot-2015-04-27";
+
+  src = fetchgit {
```

Comment:
Please use fetchFromGitHub instead, imho an archive is always better than a repo clone. The specific combo of using both fetchgit and git:// is especially bad for proxied users.